cfpb/consumerfinance.gov
# Wagtail: Adds some initial block groupings

## Additions

- Adds initial wagtail block groupings (data & research chart related blocks, expandable blocks, and wells)

## Testing

1. Pull branch, run local server and log into Wagtail.
2. Create a new browse page and add a block in the content area and see that the blocks have some groupings.

## Screenshots

<img width="645" alt="Screen Shot 2019-09-13 at 11 35 27 AM" src="https://user-images.githubusercontent.com/704760/64875367-fc2fde00-d61a-11e9-81ad-9b9a6d82ef4c.png">


## Notes
- Groups don't need migrations, so it's pretty easy to tweak this organization.

## Todos

- https://github.com/cfpb/cfgov-refresh/pull/5230 needs to be included in the well blocks group.
